filebeat/module/iis/access/_meta/fields.yml

```diff
@@ -0,0 +1,159 @@
+- name: access
+  type: group
+  description: >
+    Contains fields for IIS access logs.
+  fields:
+    - name: server_ip
+      type: keyword
+      description: >
+        The server IP address.
+    - name: method
+      type: keyword
+      example: GET
+      description: >
+        The request HTTP method.
+    - name: url
+      type: keyword
+      description: >
+        The request HTTP URL.
+    - name: query_string
+      type: keyword
+      description: >
+        The request query string, if any.
+    - name: port
+      type: long
+      description: >
+        The request port number.
+    - name: user_name
+      type: keyword
+      description: >
+        The user name used when basic authentication is used.
+    - name: remote_ip
+      type: keyword
+      description: >
+        The client IP address.
+    - name: referrer
+      type: keyword
+      description: >
+        The HTTP referrer.
+    - name: response_code
+      type: long
+      description: >
+        The HTTP response code.
+    - name: sub_status
+      type: long
+      description: >
+        The HTTP substatus code.
+    - name: win32_status
+      type: long
+      description: >
+        The Windows status code.
+    - name: time_taken_ms
```

Comment:
`resquest_time_ms` maybe?

Reply:
Agreed, I've changed this to `request_time_ms`.